packages/react-components/src/components/VideoTile.tsx

```diff
@@ -151,7 +163,47 @@ const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
+ */
+export type VideoTileMenuItem = {
+  key: string;
+  ariaLabel?: string;
+  text: string;
+  onClick: () => void;
+  icon: IIconProps;
+};
+
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
+ */
+export type VideoTileMenuItems = Array<VideoTileMenuItem>;
+
+/* @conditional-compile-remove(pinned-participants) */
+const menuIcon = (): JSX.Element => <MoreHorizontal20Filled primaryFill="currentColor" />;
+
 const defaultPersonaStyles = { root: { margin: 'auto', maxHeight: '100%' } };
+/* @conditional-compile-remove(pinned-participants) */
+/** @private */
+const VideoTileMoreOptionsButton = (props: {
+  menuItems?: Array<VideoTileMenuItem>;
+  menuStyles?: IStyle;
+}): JSX.Element => {
+  const { menuItems, menuStyles } = props;
+  if (!menuItems || menuItems.length === 0) {
+    return <></>;
+  }
+  return (
+    <DefaultButton
+      styles={concatStyleSets(menuButtonStyles, menuStyles ?? {})}
+      onRenderIcon={menuIcon}
+      menuIconProps={{ hidden: true }}
+      menuProps={{ items: mapMenuItemsToContextualMenuItems(menuItems) }}
```

Comment:
Can you set the directional hint for this context menu, from the Figma it looks like it should try be above this ellipsis:
![image](https://user-images.githubusercontent.com/2684369/201713273-ba50d00a-8c99-4307-a9c8-62fdfb3d090d.png)

Here's is the Teams behavior for reference (to the side):
![image](https://user-images.githubusercontent.com/2684369/201713311-693ccafb-f72b-48b6-bae3-887fbb16c6d1.png)

And here is example of the property from fluent's docs for reference: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu (see `ContextualMenu with beak and directional settings`)

Reply:
+1

Reply:
done